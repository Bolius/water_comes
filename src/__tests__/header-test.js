import React from "react";
import { create } from "react-test-renderer";
import ReactDOM from "react-dom";
import Header from "../components/header.js";

test("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Header />, div);
});

describe("Header component", () => {
  test("it matches the snapshot", () => {
    const component = create(<Header />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
