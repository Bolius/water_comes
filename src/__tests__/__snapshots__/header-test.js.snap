// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component it matches the snapshot 1`] = `
<div
  style={
    Object {
      "background": "url(water_header.png)",
      "backgroundPosition": "center center",
      "backgroundRepeat": "no-repeat",
      "backgroundSize": "cover",
      "color": "white",
      "height": "300px",
      "marginBottom": "10px",
      "paddingTop": "100px",
      "textAlign": "center",
    }
  }
>
  <h1
    style={
      Object {
        "fontSize": "50px",
        "fontWeight": "300",
        "textTransform": "uppercase",
      }
    }
  >
    TJEK DIN RISIKO
  </h1>
</div>
`;
